model-sync-gradle-test/build.gradle.kts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import GraphLang.L_GraphLang
+import jetbrains.mps.lang.core.L_jetbrains_mps_lang_core
+import org.modelix.model.server.Main
+
+buildscript {
+    val modelixCoreVersion: String = file("../version.txt").readText()
+    dependencies {
+        classpath("org.modelix:model-server:$modelixCoreVersion")
+        classpath("org.modelix:graph-lang-api:$modelixCoreVersion")
+    }
+}
+
+plugins {
+    kotlin("jvm")
+    id("org.modelix.model-sync")
+}
+
+val modelixCoreVersion: String = file("../version.txt").readText()
+
+version = modelixCoreVersion
+
+repositories {
+    mavenLocal()
+    maven { url = uri("https://repo.maven.apache.org/maven2") }
+    maven { url = uri("https://plugins.gradle.org/m2/") }
+    mavenCentral()
+    maven { url = uri("https://artifacts.itemis.cloud/repository/maven-mps/") }
```

Comment:
To speed up dependency resolution and to avoid leaking names of dependencies to public repositories, it would be a good idea to set up [repository content filtering](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:repository-content-filtering). Right now, Gradle would try to resolve all dependencies from our maven-mps repo first in all repos mentioned before. That takes a lot of time and also means that mavenCentral will know about all dependencies we want to request from other repos.